manifests/params.pp

```diff
@@ -11,6 +11,7 @@
   $repo_proxy_password         = 'absent'
   $repo_proxy_username         = 'absent'
   $repo_url_suffix             = '0.10'
+  $repo_release                = undef
```

Comment:
It's clearer if you default to $::lsbdistcodename here, no?

Reply:
As per https://github.com/voxpupuli/puppet-nginx/pull/1154#discussion_r151865121, changing to pass $repo_release through unaltered and let apt module deal with undef release.

Reply:
Added datatype to new parameter, and pass it straight through apt module in commits https://github.com/voxpupuli/puppet-nodejs/pull/342/commits/fdb655df9c5ddb47922c112c9f65cc757da8f473  https://github.com/voxpupuli/puppet-nodejs/pull/342/commits/63c3d3c7812f002d850f9658e7eae7f33a86fa58